### ISSUE TITLE
Searching over warm_start in SGDClassifier, not tuning learning rate

### ISSUE THREAD

**Issue by amueller** [rhiever/sklearn-benchmarks#19]
Searching over warm_start in SGDClassifier is not meaningful.
Also, using a constant learning rate and not adjusting the rate will probably not work.
the "optimal" rate will only work with scaled data.
